turbo.json

```diff
@@ -19,6 +19,11 @@
       "dependsOn": ["^build"],
       "env": ["THIRDWEB_EWS_SDK_VERSION"]
     },
+    "@thirdweb-dev/checkouts#build": {
+      "outputs": ["dist/**", "evm/**", "solana/**"],
```

Comment:
which folders does the build actually output? I assume it's just `dist/**`

Reply:
changed this